x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/EsqlFeatures.java

```diff
@@ -174,6 +174,11 @@ public class EsqlFeatures implements FeatureSpecification {
      */
     public static final NodeFeature METRICS_SYNTAX = new NodeFeature("esql.metrics_syntax");
 
+    /**
+     * Supports repeat string function
+     */
+    public static final NodeFeature REPEAT = new NodeFeature("esql.repeat");
```

Comment:
In the time since you started all this we moved to adding things to `EsqlCapabilities`. Could you move this one there?